docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG branch=latest
-FROM cccs/assemblyline:$branch
+FROM cccs/assemblyline-core:$branch
```

Comment:
Shouldn't this be fixed in the pipeline instead? Are there extra thing that we don't want from the core container?

Reply:
Not really. Everything that is in the core container, gets added in this one.